<?xml version="1.0"?>
<launch>
	
	<!-- Eye_tracker arguments -->
		<arg name="model_path" 	 default='/home/paolo/rosneuro_ws/src/eye_tracking/trained_models/shape_predictor_68_face_landmarks.dat'/>

	<!-- aquisition node -->
	<node name="gaze_node" pkg="eye_tracking" type="gaze_node" output="screen" >
		<param name="~model_path" 	  value="$(arg model_path)"/>
	</node>

</launch>
